python/hawkey/nevra-py.c

```diff
@@ -220,7 +220,9 @@ evr_cmp(_NevraObject *self, PyObject *args)
 static PyObject *
 has_just_name(_NevraObject *self, PyObject *unused)
 {
-    return hy_nevra_has_just_name(self->nevra) ? Py_True : Py_False;
+    if (hy_nevra_has_just_name(self->nevra))
+        Py_RETURN_TRUE;
+    Py_RETURN_FALSE;
```

Comment:
FWIW, this could also be spelled `return PyBool_FromLong(hy_nevra_has_just_name(self->nevra));`